.github/workflows/ros_translation_node.yml

```
@@ -0,0 +1,45 @@
+name: ROS translation node tests
+on:
+  push:
+    branches:
+    - 'main'
+  pull_request:
+    branches:
+    - '*'
+defaults:
+  run:
+    shell: bash
+jobs:
+  build_and_test:
+    name: Build and test
+    runs-on: ubuntu-latest
```

Given how this build is taking longer than 5min I suggest and running on every PR + push to main, I suggest we move this to our AWS infra

```suggestion
    runs-on: [runs-on,runner=8cpu-linux-x64,image=ubuntu24-full-x64,"run-id=${{ github.run_id }}",spot=false]
```

---

Thanks, I updated that